swagger.json

```diff
@@ -4014,40 +4014,97 @@
                     "type": "string",
                     "description": "Reason that this driver is exempt from the Electronic Logging Mandate (see eldExempt)."
                 },
-                "eldBigDayExemptionEnabled": {
+                "usBigDayExemptionEnabled": {
```

Comment:
We will need to duplicate these fields and leave the existing ones to not break current implementations.

Reply:
I'll leave the existing ones there in our actual API implementation, but delete them here in the documentation.